### ISSUE TITLE
Allow roles export to be beautified (pretty print)

### ISSUE THREAD

**Issue by EBoisseauSierra** [dpgaspar/Flask-AppBuilder#1716]
Running `export-roles` creates a one-line only JSON file. To generate meaningful diffs when VCS-ing the export, I would like it to be beautified/pretty printed.

### Describe the expected results

I would like the JSON export looks like this:

```json
{
    "a": 0, 
    "b": 0, 
    "c": 0
}
```

### Describe the actual results

Current export looks like this:

```json
{"a": 0, "b": 0, "c": 0}
```

**Comment by dpgaspar** [comment]
sounds good to me, thank you!

**Comment by EBoisseauSierra** [comment]
Has been fixed by #1724 .
